<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.1.1 (Python 3.13.0 on win32)" generated="2024-12-11T15:29:21.118270" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\beimnet.bekele\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2024-12-11T15:29:21.181810" elapsed="5.053658"/>
</kw>
<status status="PASS" start="2024-12-11T15:29:21.181223" elapsed="5.054699"/>
</test>
<status status="PASS" start="2024-12-11T15:29:21.122848" elapsed="5.114659"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2024-12-11T15:29:21.106627" level="WARN">Error in file 'c:\Users\beimnet.bekele\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
